app/concepts/user/create.rb

```diff
@@ -0,0 +1,19 @@
+class User::Create < Trailblazer::Operation
+  include Model
+  model User, :create
+
+  contract do 
+    property :fullname
+    validates :fullname, presence: true
+  end
+  
+  def process(params)
+    params[:todo][:fullname] = params[:todo][:fullname].blank? ? "Guest" : params[:todo][:fullname]
+    validate(params[:todo],@model) do |f|
```

Comment:
good job! this `validate` hook in `process` method is super important. You could use the same behaviour on `TodoList` crud. I also noticed that you didn't validate properties on `TodoList`. The model is worth validating, say... a `TodoList` that doesn't have a `name`? How's that?